Source/DafnyCore/Dafny.atg

```diff
@@ -3934,16 +3934,21 @@ LogicalExpression<out Expression e0, bool allowLemma, bool allowLambda, bool all
       AndOp                                                     (. x = t; .)
       RelationalExpression<out e1, allowLemma, allowLambda, allowBitwiseOps>
          (. startToken = e0.StartToken;
-            e0 = new BinaryExpr(x, BinaryExpr.Opcode.And, e0, e1, firstOp);
-            e0.RangeToken = new RangeToken(startToken, t); .)
+            var fmtToken = e0.RangeToken.EndToken.Next;
+            e0 = new BinaryExpr(x, BinaryExpr.Opcode.And, e0, e1);
+            e0.RangeToken = new RangeToken(startToken, t);
+            e0.FormatTokens = new[] { fmtToken };
```

Comment:
Isn't it true that fmtToken == x ?
In this case, i might be useless to store the format token anyway.

Reply:
You may be right. It's possible that this can be simplified even further.

Reply:
Oh, of course. Instead of `fmtToken` it should be `firstOp`, as it was when `firstOp` was passed into the constructor.